test_logwarn

```diff
@@ -89,7 +89,7 @@ is-ignored '\[.*warn\] Unable to get filename'
 is-ignored '\[.*warn\] has no child, check its machine placed'
 is-ignored '\[error\] DBIx.*number of parameters must be between.*OpenQA/WebAPI/Controller/API/V1/Job.pm'
 is-ignored '\[error\] mkdir /var/lib/openqa/images/0bf/862/.thumbs: File exists at /usr/share/openqa/script/../lib/OpenQA/Schema/Result/Jobs.pm line 1227'
-is-ignored '\[warn\].*Ignoring invalid group {"name":".*"}'
+is-ignored '\[warn\].*Ignoring invalid group '
```

Comment:
Shouldn't we still keep this specific to confirm that the more specific message was ignored?

Reply:
In the test it can't harm to be less specific, while being too specific brings the risk that we are trying to match something that's not in the test log at all.